src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/RemoteDebuggerAuthenticationService.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.ProjectSystem.Debug;
+using Microsoft.VisualStudio.Shell.Interop;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.Debug
+{
+    [Export(typeof(IRemoteDebuggerAuthenticationService))]
+    [AppliesTo(ProjectCapabilities.AlwaysApplicable)]
+    internal class RemoteDebuggerAuthenticationService : IRemoteDebuggerAuthenticationService
```

Comment:
I think these should be applied with [Order] which is respected so that we have a predictable order in which they are displayed in the UI (ie None, Windows, Universal), currently its based on the order that MEF returns them.